drizzle/schema.ts

```diff
@@ -120,3 +120,17 @@ export const files1 = pgTable("files1", {
   url: varchar("url", { length: 255 }),
   type: varchar("type", { length: 255 }),
 });
+
+export const otps1 = pgTable("otps1", {
+  id: serial("id").primaryKey().notNull(),
+  phoneNumber: varchar("phoneNumber", { length: 32 }).notNull(),
+  otp: varchar("otp", { length: 255 }).notNull(),
+  createdAt: timestamp("createdAt", {
+    withTimezone: true,
+    mode: "string",
+  }).defaultNow(),
```

Comment:
As this is a duration rather than an explicit timepoint, maybe it's actually better to not save the timezones as the unix timestamp would be the same globally regardless of where you were on the earth.

I'm a bit curious about if the database and the server are in different time zones, but avoid this, maybe we just call `new Date()` once in `sendOtpViaWhatsApp` and use that value for both the `createdAt` time and +10 for the `expiresAt` time.